host/3.0/buster/amd64/node/node10/node10-composite.template

```diff
@@ -0,0 +1,42 @@
+## Removed find /workers/node/grpc/src/node/extension_binary/ -mindepth 1 -type d ! -regex ".*linux-x64.*" -prune -exec rm -rf '{}' \;
+# This line removes grpc stuff built for unsupported x64 builds. Can probably delete from main host dockerfile.  need to inventory 
+
+FROM mcr.microsoft.com/dotnet/core/runtime-deps:3.1
+ARG HOST_VERSION
+
+RUN apt-get update && \
+    apt-get install -y curl gnupg && \
+    curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
+    apt-get update && \
+    apt-get install -y nodejs
+
+# Chrome headless dependencies
+# https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#chrome-headless-doesnt-launch-on-unix
```

Comment:
Can installing `Chrome headless dependencies` move to host.dockerfile?

Reply:
Unsure, only because as far as I understand node and python are the only ones that have it.  I will investigate what I can do to consolidate this. 
https://github.com/Azure/azure-functions-docker/pull/417

You also made a comment in this PR to see if these could be moved to the main dockerfile.

Reply:
Thanks for linking earlier PR - makes sense to move this to common file as it does not seem to language specific - we can track this separately as well. 

We should move this to host.dockerfile as long as it does not increase image size drastically as Node and Python images need it anyways